### DISCUSSION TITLE
Plantilla de Buyer persona

### DISCUSSION THREAD

**Discussion by CesarMezaB** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#55] [Ideas]
Les comparto la siguiente página de donde se saco el ejemplo de las practica para que generen el archivo de buyer persona y les de también una referencias de lo que pueden poner en cada uno de los campos
https://www.hubspot.es/make-my-persona

**Comment by JuanSalvador73** [comment]
Excelente información, gracias.
